src/admin/paths/admin@spend_overview@summary.yaml

```diff
@@ -0,0 +1,49 @@
+get:
+  tags:
+    - Spend Overview
+  summary: Spend Overview Summary
+  operationId: spend_overview_summary
+  description: |
+    API to return spend overview summary data. Data returned by this API can be used to plot a time series graph depicting amount of spend over a period of time. <br>
+    This API supports very rich filtering on all response fields and pagination via query parameters. To understand how to use these parameters, please see the [Guide to Data APIs](https://docs.fylehq.com/docs/fyle-platform-docs/ZG9jOjI3Mzk2NTM3-guide-to-data-ap-is#filtering)
+  parameters:
+    - $ref: ../../components/parameters/time_interval_granularity.yaml
+    - $ref: ../../components/parameters/time_interval_preset.yaml
+    - $ref: ../../components/parameters/time_interval_start.yaml
+    - $ref: ../../components/parameters/time_interval_end.yaml
+    # - $ref: ../../components/parameters/is_card_spend.yaml  # will un-comment later when this filter is available
+    - $ref: ../../components/parameters/is_reimbursable.yaml
+    - $ref: ../../components/parameters/merchants.yaml
+    - $ref: ../../components/parameters/category_ids.yaml
```

Comment:
why are these parameters plural? they should all be singular right?

client will be sending `category_id=in.(c1, c2)`